resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/shuffle-service/Dockerfile

```diff
@@ -15,25 +15,12 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-alpine
+FROM spark-base
```

Comment:
ideally these would have version tags like `2.1.0-kubernetes-0.3.0` rather than something like `latest` (I'm a fan of immutable artifacts)

We'd need to have some way of templating Dockerfiles to do that though I think